### PR TITLE
vcrun2019: update hashes

### PR DESCRIPTION
**Pull request by x0nn**
Hash changed and so this modification is needed

### CODE REVIEW

File: src/winetricks [hunk 1]
```diff
@@ -12885,7 +12885,9 @@ load_vcrun2019()
     # 2020/10/03: caa38fd474164a38ab47ac1755c8ccca5ccfacfa9a874f62609e6439924e87ec
     # 2020/11/13: 50a3e92ade4c2d8f310a2812d46322459104039b9deadbd7fdd483b5c697c0c8
     # 2021/03/09: 4521ed84b9b1679a706e719423d54ef5e413dc50dde1cf362232d7359d7e89c4
-    w_download https://aka.ms/vs/16/release/vc_redist.x86.exe 4521ed84b9b1679a706e719423d54ef5e413dc50dde1cf362232d7359d7e89c4
+    # 2021/03/28: e830c313aa99656748f9d2ed582c28101eaaf75f5377e3fb104c761bf3f808b2
+
```
Comment (gverm) [inline src/winetricks:12889]: This whitespace change is unnecessary.

File: src/winetricks [hunk 2]
```diff
@@ -12909,11 +12911,12 @@ load_vcrun2019()
             # 2020/10/03: 4b5890eb1aefdf8dfa3234b5032147eb90f050c5758a80901b201ae969780107
             # 2020/11/13: b1a32c71a6b7d5978904fb223763263ea5a7eb23b2c44a0d60e90d234ad99178
             # 2021/03/09: f299953673de262fefad9dd19bfbe6a5725a03ae733bebfec856f1306f79c9f7
+            # 2021/03/09: b6c82087a2c443db859fdbeaae7f46244d06c3f2a7f71c35e50358066253de52
```
Comment (gverm) [inline src/winetricks:12914]: The date here is wrong. (copy & paste error)

**Review by gverm — CHANGES_REQUESTED** [review]
Some comments.

File: src/winetricks (hunk 1 above)
Comment (x0nn) [reply src/winetricks]: removed...

File: src/winetricks (hunk 2 above)
Comment (x0nn) [reply src/winetricks]: corrected...

**Comment by gverm** [comment]
@x0nn Travis seems to imply there is added trailing whitespace somewhere, could you have a look?

**Pull request merged**